multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=0.7.15
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO := git
+PKG_SOURCE_URL := https://github.com/catalinii/minisatip.git
+PKG_SOURCE_VERSION := fdbf44fc48392b0735e0bf17b31aaff8f06740d0
+
+include $(INCLUDE_DIR)/package.mk
```

Comment:
Please add yourself as the `PKG_MAINTAINER`.  You may also want to declare yourself as the copyright holder in the file header.